### ISSUE TITLE
Add support for DASH output

### ISSUE THREAD

**Issue by JustJoostNL** [membraneframework/membrane_core#807]
Hey, 
Would it be possible to add support for DASH output?

Thanks.

**Comment by mat-hek** [comment]
Hi @JustJoostNL, we planned to support DASH, but it turned out to have no significant advantages over HLS in the use cases we were involved in. Also, DASH isn't supported on some Apple devices (though that changes slowly AFAIK). Can you share why you chose DASH over HLS?

**Comment by JustJoostNL** [comment]
Hey @mat-hek,

We prefer DASH over HLS in our use case because of the following:
- More stable (for us at least)
- Efficient fragmentation
- Better for live streaming (easier to set a target latency)
- Rich metadata support
- Better handling of variable frame rates

**Comment by JustJoostNL** [comment]
With efficient fragmentation I mean that DASH allows more efficient use of HTTP/2 and QUIC, leading to potentially lower latency and better performance over new transport protocols.

We use a combination of both CMAF and MPEG-TS, though mostly MPEG-TS currently.

**Comment by mat-hek** [comment]
Ok, so since we use CMAF for HLS, CMAF over DASH wouldn't be hard (I guess), so we can consider doing that in the foreseeable future. However, especially regarding efficiency, I'd rather lean towards the Media Over QUIC effort, see https://github.com/kixelated/moq-rs

**Comment by JustJoostNL** [comment]
@mat-hek Yes, that seems good. Also, another question: it is not currently possible to use MPEG-TS with HLS in Membrane right?

**Comment by mat-hek** [comment]
> Yes, that seems good

So we plan to research that. Contributions are welcome in this area ;)

> it is not currently possible to use MPEG-TS with HLS in Membrane right?

Right. There's a third-party implementation of HLS/MPEG-TS ingest though https://github.com/kim-company/membrane_hls_plugin

**Comment by JustJoostNL** [comment]
Would love to contribute, though, unfortunately my knowledge isn’t good enough for this.
